pkg/v1/tkg/client/cluster.go

```diff
@@ -214,6 +208,30 @@ func (c *TkgClient) waitForClusterCreation(regionalClusterClient clusterclient.C
 	return nil
 }
 
+// WaitForAutoscalerDeployment waits for autoscaler deployment if enabled
+func (c *TkgClient) WaitForAutoscalerDeployment(regionalClusterClient clusterclient.Client, clusterName, targetNamespace string) {
+	var autoscalerEnabled string
+	var err error
+
+	if autoscalerEnabled, err = c.TKGConfigReaderWriter().Get(constants.ConfigVariableEnableAutoscaler); err != nil {
+		return
+	}
+
+	var isEnabled bool
+	if isEnabled, err = strconv.ParseBool(autoscalerEnabled); err != nil {
+		log.Warningf("Unable to parse the value of config variable %q. reason: %v", constants.ConfigVariableEnableAutoscaler, err)
+		return
+	}
```

Comment:
This can be another helper method that returns `bool`.

Reply:
Done!

Reply:
Sorry, I should have made my point more clear. I was talking about the whole piece of code beginning with `var autoscalerEnabled string` declaration. Also, just noticed, we're swallowing the error doing this:
```go
	if autoscalerEnabled, err = c.TKGConfigReaderWriter().Get(constants.ConfigVariableEnableAutoscaler); err != nil {
		return
	}
```